homeassistant/components/airzone/climate.py

```diff
@@ -128,13 +127,33 @@ async def _async_update_hvac_params(self, params) -> None:
         """Send HVAC parameters to API."""
         try:
             await self.coordinator.airzone.put_hvac(params)
-        except (AirzoneError, ClientConnectorError) as error:
+        except AirzoneError as error:
             raise HomeAssistantError(
                 f"Failed to set zone {self.name}: {error}"
             ) from error
         else:
             self.coordinator.async_set_updated_data(self.coordinator.airzone.data())
 
+    async def async_turn_on(self) -> None:
+        """Turn the entity on."""
+        params = {
+            API_SYSTEM_ID: self.system_id,
+            API_ZONE_ID: self.zone_id,
```

Comment:
In a future code quality PR you can add a method `_set_params`, something like this:
```python
    def _set_params(self, params: dict[str, Any]) -> None:
        _params = {API_SYSTEM_ID: self.system_id, API_ZONE_ID: self.zone_id, **params}
        _LOGGER.debug("set_params=%s", _params)
        await self._async_update_hvac_params(_params)
```

Reply:
Done :)
https://github.com/home-assistant/core/pull/70099